content/hackerstoolbox/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: About Hackers Toolbox
+description: About Hackers Toolbox
+date: "2023-06-23"
+---
+
```

Comment:
add some pics maybe?

Reply:
No media from past sessions really suitable to use for a banner image, will upload one once we have our first session this semester and take some pics.